person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :name, :age, :id
+
```

Comment:
- [ ] Kindly remove `:id` from `attr_accessor` and add it to `attr_reader` instead. This is because `:id` should only be able to be retrieved and not set.